README.md

```diff
@@ -164,33 +161,31 @@ We also deliver an **out-of-the-box web interface** in which you can test MeiliS
 
 You can access the web interface in your web browser at the root of the server. The default URL is [http://127.0.0.1:7700](http://127.0.0.1:7700). All you need to do is open your web browser and enter MeiliSearch’s address to visit it. This will lead you to a web page with a search bar that will allow you to search in the selected index.
 
-<p align="center">
-  <img src="assets/movies-web-demo.gif" alt="Web interface gif" />
-</p>
+| [See the gif above](#demo)
 
-### Documentation
+## Documentation
 
 Now that your MeiliSearch server is up and running, you can learn more about how to tune your search engine in [the documentation](https://docs.meilisearch.com).
 
-
 ## Contributing
 
 Hey! We're glad you're thinking about contributing to MeiliSearch! If you think something is missing or could be improved, please open issues and pull requests. If you'd like to help this project grow, we'd love to have you! To start contributing, checking [issues tagged as "good-first-issue"](https://github.com/meilisearch/MeiliSearch/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) is a good start!
 
-### Analytic Events
+## Telemetry
 
-Every hour, events are being sent to our Amplitude instance so we can know how many people are using MeiliSearch.<br/>
+MeiliSearch collects anonymous data regarding general usage.
+This helps us better understand developers usage of MeiliSearch features.<br/>
 To see what information we're retrieving, please see the complete list [on the dedicated issue](https://github.com/meilisearch/MeiliSearch/issues/720).<br/>
 We also use Sentry to make us crash and error reports. If you want to know more about what Sentry collects, please visit their [privacy policy website](https://sentry.io/privacy/).<br/>
-If this doesn't suit you, you can disable these analytics by using the `MEILI_NO_ANALYTICS` env variable.
+This program is optionnal, you can disable these analytics by using the `MEILI_NO_ANALYTICS` env variable.
```

Comment:
What do you mean by "this program", this env variable disable all the analytics, sentry and our own analytics.

Reply:
The telemetry program.